scripts/cubic/bootstrap-fmriprep.sh

```diff
@@ -218,41 +209,43 @@ chmod +x code/participant_job.sh
 cat > code/fmriprep_zip.sh << "EOT"
 #!/bin/bash
 set -e -u -x
-
 subid="$1"
 mkdir -p ${PWD}/.git/tmp/wdir
+# TODO: fix path to singularity image
 singularity run --cleanenv -B ${PWD} \
-    pennlinc-containers/.datalad/environments/fmriprep-20-2-3/image \
+    containers/images/bids/bids-fmriprep--20.2.3.sing \
```

Comment:
we will finalize this command after everything is set. but this is the command that may need to be provided by the user as a template of some kind to generalize across different use cases.